WordPress/create-block-theme
# Copy font assets to the local theme folder when creating a style variation

## What?
Copy font assets to the local theme folder when creating a style variation.

## Why?
Fixes: https://github.com/WordPress/create-block-theme/issues/695

## How to test:
- Create a blank theme
- Install a few font families using the font library
- Create a variation with those changes and the 'save fonts' option selected.
- Verify that the fonts were copied to the local theme folder.

## Screencast
https://github.com/user-attachments/assets/921bcd99-dac3-42ab-9dea-6860c095bd3d
